etc/schema.json

```diff
@@ -1156,7 +1156,12 @@
                         "opcode": {
                             "description": "DNS opcode as an integer",
                             "type": "integer"
-                        }
+                        },
+			"authorities": {
+			    "description": "Array of authorities in DNS response",
+			    "$comment": "Probably also need answers and additional records",
+			    "type": "array"
+			}
                     },
```

Comment:
Is this still what we need or is it https://github.com/OISF/suricata/pull/10147/commits/6286be1cbed2ad219a469d5c78840a8ec973465c @jasonish ?

Reply:
No, we'll now want the commit from https://github.com/OISF/suricata/pull/10147.